plugins/content/local/writer.go

```diff
@@ -24,25 +24,29 @@ import (
 	"os"
 	"path/filepath"
 	"runtime"
+	"sync"
 	"time"
 
 	"github.com/containerd/containerd/v2/core/content"
+	"github.com/containerd/containerd/v2/pkg/fsverity"
 	"github.com/containerd/errdefs"
 	"github.com/containerd/log"
 	"github.com/opencontainers/go-digest"
 )
 
 // writer represents a write transaction against the blob store.
 type writer struct {
-	s         *store
-	fp        *os.File // opened data file
-	path      string   // path to writer dir
-	ref       string   // ref key
-	offset    int64
-	total     int64
-	digester  digest.Digester
-	startedAt time.Time
-	updatedAt time.Time
+	s                  *store
+	fp                 *os.File // opened data file
+	path               string   // path to writer dir
+	ref                string   // ref key
+	offset             int64
+	total              int64
+	digester           digest.Digester
+	startedAt          time.Time
+	updatedAt          time.Time
+	once               sync.Once
+	integritySupported bool
```

Comment:
what about moving this to `store` struct?

```go
type store struct {
      root string
      ls LabelStore
      
      integritySupported bool
}
```

So that we can just run fsverity.IsSupported once and all the writers from same store can share it.

Reply:
Thank you, that makes more sense now. I've made the change.